server/channels/store/storetest/user_store.go

```diff
@@ -6169,3 +6170,14 @@ func testGetUsersWithInvalidEmails(t *testing.T, ss store.Store) {
 	require.NoError(t, err)
 	assert.Len(t, users, 1)
 }
+
+func testUpdateLastLogin(t *testing.T, ss store.Store) {
+	u1 := model.User{}
+	u1.Email = MakeEmail()
+	_, err := ss.User().Save(&u1)
+	require.NoError(t, err)
+	defer func() { require.NoError(t, ss.User().PermanentDelete(u1.Id)) }()
+
+	err = ss.User().UpdateLastLogin(u1.Id, 1234567890)
```

Comment:
This test seems to be testing that there is no error, but it doesn't seem to be testing the value is properly assigned. Are we trusting the implementation underneath, or do we want to add that to this test?

Reply:
We can add that to the test.